test/integration/pod_shutdown_test.go

```diff
@@ -120,7 +120,7 @@ func testShutdownAllPodsOfApp(t *testing.T, env *testlib.TestEnv, namespace stri
 	// The leader election lease should already contain the empty string as the holder, because the old leader
 	// pod should have given up the lease during its graceful shutdown.
 	waitForLeaderElectionLeaseToHaveHolderIdentity(t, namespace, appName,
-		func(holder string) bool { return holder == "" }, 200*time.Millisecond)
+		func(holder string) bool { return holder == "" }, 1*time.Minute)
```

Comment:
5 times per second down to once a minute, quite a bit longer?

Reply:
1 minute is how to to wait for the predicate to become true (not how often to poll during that minute).